package.json

```diff
@@ -27,7 +26,7 @@
     "docusaurus-plugin-sass": "^0.2.1",
     "file-loader": "^6.2.0",
     "mdx-mermaid": "^1.2.2",
-    "mermaid": "^8.14.0",
+    "mermaid":"8.11.0",
```

Comment:
Why change to lower version?

Reply:
> warning " > mdx-mermaid@1.2.3" has incorrect peer dependency "mermaid@>= 8.11.0 < 8.12.0".
I'm not sure if this change is necessary.

Reply:
OK